### PR TITLE
[NOTASK]: Update styling of Home > Success Stories

### PR DESCRIPTION
**Pull request by eunicey**
*I made sure to not change any of the updates made in [[NOTASK]: unify gutter spacing, header spacing](https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/pull/39/files#top) 
~~with the exception of one (see comments) which can be easily added back if necessary.~~

Updated styling of Success Stories
- spacing between elements especially at desktop width
- size of image
- font size of author name

BEFORE:
BEFORE
AFTER:
AFTER

### CODE REVIEW

File: src/components/HomeContainer/SuccessStories/SuccessStories.module.css [hunk 1]
```diff
@@ -78,7 +79,6 @@
 @media (min-width: 960px) {
   .wrapper {
     padding-block: 140px;
-    max-width: 1148px;
```
Comment (eunicey) [inline src/components/HomeContainer/SuccessStories/SuccessStories.module.css:81]: This is the only edit to what @joanwolk added in the previous PR. It includes the 165px L/R padding so the content width is quite narrow for desktop.

Comment (eunicey) [reply src/components/HomeContainer/SuccessStories/SuccessStories.module.css:81]: Ignore my previous comment. In my latest commit, I've added max-width back but made the box-sizing to content-box.

**Comment by joanwolk** [comment]
Using content-box breaks displays for small desktops, where the stories overflow the content area and also the viewport:
Screenshot 2024-08-01 at 9 53 48 AM

Using border-box and putting the max-width on `.contents` instead of `.wrapper` will get things _mostly_ working, although the names will still misalign at some sizes. The solution for that is some work with `display: grid` I think, although I'm having some trouble executing the thing that should work.

**Comment by eunicey** [comment]
@joanwolk - Thanks for the suggestion. I made the change you recommended for the max-width  and box-sizing. I won't have the time today to explore the display grid solution though.

**Comment by joanwolk** [comment]
I solved my grid issue!
- `.contents` grid sets tidy columns, and aligns child items to the end of the block
- `.quote` is fixed to full height and uses rows of 1fr to set height of child blocks smartly
- .`quote > :first-child` sets the classless first child of `.quote` to align at the start of the grid instead of the end, so the extra whitespace is after the quotation and before the name (and I wouldn't have to change a snapshot if one were present, as I would if I added a class)

**Review by joanwolk — APPROVED** [review]

File: src/components/HomeContainer/SuccessStories/SuccessStories.module.css [hunk 2]
```diff
@@ -29,14 +28,14 @@
...
-  background: transparent url("../../../assets/icon-quotes.svg") no-repeat center top / 56px 56px;
+  padding-top: 36px;
+  background: transparent url("../../../assets/icon-quotes.svg") no-repeat center top / 67px 56px;
 }
 
 /* image */
 .image {
-  width: 60px;
-  height: 60px;
+  width: 66px;
```
Comment (poulami-saha) [inline src/components/HomeContainer/SuccessStories/SuccessStories.module.css:37]: The width and height of each image is 62px across all screen size, as per the design, can we have a quick look at it?

Comment (eunicey) [reply src/components/HomeContainer/SuccessStories/SuccessStories.module.css:37]: @joanwolk - I included the 2px border which is not accounted for in the design when you click on the image size. I did the same for the image on the profile cards.

**Pull request merged**